addons/mail/static/src/rtc/call_participant_card.xml

```diff
@@ -36,6 +36,9 @@
                     </small>
                 </span>
                 <div class="o-mail-CallParticipantCard-overlay position-absolute top-0 end-0 d-flex flex-row-reverse">
+                    <span t-if="rtcSession.isRaisingHand" class="d-flex flex-column justify-content-center me-1 rounded-circle bg-500" t-att-class="{'o-minimized p-1': props.minimized, 'p-2': !props.minimized }" title="raising hand" aria-label="raising hand">
```

Comment:
`o-minimized` looks unused (some other occurrences in this template are also unused it seems)

Reply:
@alexkuhn it used to have the `p-1` equivalent in scss and I think that the design team kept the class for backwards compatibility when they refactored

Reply:
Then can remove it, if unused then not worth keeping it ;)